src/include/compat.h

```diff
@@ -15,6 +15,7 @@
 #include "acconfig.h"
 
 #if defined(__linux__)
+#include <fnctl.h>
```

Comment:
see https://github.com/ceph/ceph/pull/20398#discussion_r180768420

Reply:
@tchaikov 
missed that remark